lib/modules.nix

```
@@ -461,9 +487,13 @@ rec {
         imports = m.require or [] ++ m.imports or [];
         options = {};
         config = addFreeformType (removeAttrs m ["_file" "key" "disabledModules" "require" "imports" "freeformType"]);
```

```suggestion
        config = addFreeformType (removeAttrs m ["_file" "key" "disabledModules" "require" "imports" "freeformType" "class"]);
```

Otherwise you can't do `imports = [ { class = "nixos" } ]`

---

If removed `class` as a keyword from the shorthand syntax. I think we want to keep that syntax for "data-like" usage and not "module-like" usage. Stealing `class` might have caused some breakage.